generated/nifgen/session.py

```diff
@@ -2581,8 +2581,8 @@ def close(self):
 
     ''' These are code-generated '''
 
-    def _abort_generation(self):
-        '''_abort_generation
+    def abort_generation(self):
```

Comment:
This is the counterpart to a public method called initiate(), not initiate_generation().
So the public name should be abort().